llvm/tools/llvm-exegesis/lib/BenchmarkResult.cpp

```diff
@@ -278,6 +287,13 @@ template <> struct ScalarTraits<exegesis::RegisterValue> {
   static const bool flow = true;
 };
 
+template <> struct ScalarEnumerationTraits<compression::Format> {
```

Comment:
I don't think we're supposed to create these for types that aren't owned by us (or at least that's what I was told when I attempted something similar). This might need to go with where the type is defined.

Reply:
Fixed.